code/game/objects/items.dm

```diff
@@ -730,21 +731,25 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 	user.visible_message("<span class='notice'>[user] peers through \the [zoom_device].</span>",
 	"<span class='notice'>You peer through \the [zoom_device].</span>")
 	zoom = TRUE
+	RegisterSignal(user, COMSIG_ITEM_ZOOM, .proc/zoom_check_return)
 	onzoom(user)
-	if(user.interactee)
-		user.unset_interaction()
-	else if(!istype(src, /obj/item/attachable/scope))
-		user.set_interaction(src)
```

Comment:
Why are you deleting this

Reply:
It has not much use and actuslly was a request by the guy making the hmg stuff, because it stops the deployed guns from being able to use scope(because the deployed guns use interactee) and i dont think scopes should use interactee because you keep scoped while shooting and stuff on guns, youre not like using both your hands to look on the scope only and not able to do something else.

Reply:
ok if it breaks anything i blame you